src/components/Calculator.js

```diff
@@ -1,41 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 import '../style.css';
 import KeyBtn from './Kyes';
-// eslint-disable-next-line react/prefer-stateless-function
-class Calc extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: 0,
-      operation: null,
-      next: null,
-    };
-  }
 
-  EventHandler = (element) => {
+const Calc = () => {
+  const [state, setState] = useState({
+    total: null,
+    operation: null,
+    next: null,
+  });
```

Comment:
- [ ] On the calculator page the calculator should be aligned to the left of the page as in the given wireframe, kindly fix ir.

Given design
![image](https://user-images.githubusercontent.com/81932478/200129558-0ff8a2b3-06d4-4957-b64e-b8052348f5aa.png)


Your design
![image](https://user-images.githubusercontent.com/81932478/200129615-57d39e35-5720-4a52-a111-3d5138cf32d2.png)